rpa-builder/6.3/modules/ROOT/pages/toolbox-variable-handling-append-to-array.adoc

```diff
@@ -0,0 +1,32 @@
+= Append to Array 
+
+Use this Action Step to append a value to an existing Array variable. 
+
+== Properties
```

Comment:
Diane and I were just talking about adding context to all headings for SEO. Maybe it's not applicable in this case, since it's a reference topic?